chia/consensus/block_body_validation.py

```diff
@@ -44,6 +44,21 @@
 #           :
 
 
+@dataclass(frozen=True)
```

Comment:
should we move this to types ?

Reply:
maybe. I'm not sure what reasoning we use to decide whether a type belongs in `chia.types` or not. `BlockRecord` isn't in types for instance, which is a bit surprising.